### PR TITLE
fixes issues with chakra plugin config

### PR DESCRIPTION
**Pull request by utkarsh867**

### CODE REVIEW

File: src/components/home/jumbotron.tsx [hunk 1]
```diff
@@ -20,22 +20,22 @@ export const Jumbotron = () => {
...
             borderRadius={0}
             mr={3}
-            _hover={{ bg: customTheme.colors.secondary }}
+            _hover={{ backgroundColor: "secondary" }}
           >
             Join Us
           </Button>
           <Button
             padding="8px 32px"
             background={customTheme.colors.transparent}
```
Comment (ahsan8244) [inline src/components/home/jumbotron.tsx]: let's change this to background-color as well

File: src/components/layout.tsx [hunk 2]
```diff
@@ -1,15 +1,11 @@
 import React from "react";
 import { Flex, Text } from "@chakra-ui/core";
 import styled from "styled-components";
-import { customTheme } from "../theme";
 import { Link as GatsbyLink } from "gatsby";
 import { globalHistory } from "@reach/router";
 
 const Nav = styled(Text)<{ route: string }>`
-  color: ${({ route }) =>
```
Comment (ahsan8244) [inline src/components/layout.tsx:9]: what about the selected nav item?

Comment (utkarsh867) [reply src/components/layout.tsx:9]: Yeah right I made a very quick change to make things work. Ended up removing all of this code. Gonna put this right back.

Comment (utkarsh867) [reply src/components/layout.tsx:9]: Ah yes, let me add this code right back in. I just wanted to get it to work at that time. Forgot to revert this change.

File: src/components/home/jumbotron.tsx (hunk 1 above)
Comment (utkarsh867) [reply src/components/home/jumbotron.tsx]: Done

**Review by ahsan8244 — APPROVED** [review]
LGTM! 👍

**Comment by utkarsh867** [comment]
I'm happy with this.

**Comment by utkarsh867** [comment]
@ahsan8244 You can go ahead and merge :)

**Pull request merged**